mathics/builtin/linalg.py

```diff
@@ -761,7 +757,6 @@ class MatrixPower(Builtin):
      = {{169, -70}, {-70, 29}}
 
     #> MatrixPower[{{0, x}, {0, 0}}, n]
```

Comment:
Is the result givne by sympy correct?
Mathematica seems to give something like 

![image](https://user-images.githubusercontent.com/885054/59086743-ebc4e480-8902-11e9-894d-311e4683324e.png)

Reply:
If n is negative, then there's no valid result since the matrix is not invertible. So sympy returns the expression as is.